src/client/containers/layout.js

```diff
@@ -239,11 +240,11 @@ class Layout extends React.Component {
 		);
 
 		const alerts = this.props.alerts.map(
-			(alert, idx) => (
+			(alert) => (
 				<Alert
 					bsStyle={alert.level}
 					className="text-center"
-					key={idx}
+					key={alert.id}
```

Comment:
`alert.id` does not exist. See for example https://github.com/bookbrainz/bookbrainz-site/blob/fc9881584ae896c1dde6aae4ef98b422f40fc6f8/src/server/app.js#L131-L135

You will need to put the `idx` index back, so that you can do `` key={`alert-${idx}`} ``